charts/steward/crds/pipelineruns.yaml

```diff
@@ -78,6 +78,9 @@ spec:
                 - run
                 - abort
                 default: run
+              "timeout": ###
+                type: string
+                pattern: '^[0-9]*m?$'
```

Comment:
We shouldn't introduce another specification different from that of [`time.ParseDuration()`](https://pkg.go.dev/time#ParseDuration). An exception might be that negative durations are not allowed.

Reply:
Modified the regex to: ([0-9]+h)?([0-9]+m)?([0-9]+s)?, this allows setting the timeout in hrs, min, and secs